ush/forecast_postdet.sh

```diff
@@ -536,52 +542,57 @@ MOM6_postdet() {
   fi  # if [[ "${RERUN}" == "NO" ]]; then
 
   # Link output files
-  if [[ "${RUN}" =~ "gfs" || "${RUN}" == "gefs" ]]; then  # Link output files for RUN=gfs|enkfgfs|gefs
-
-    # Looping over MOM6 output hours
-    local fhr fhr3 last_fhr interval midpoint vdate vdate_mid source_file dest_file
-    for fhr in ${MOM6_OUTPUT_FH}; do
-      fhr3=$(printf %03i "${fhr}")
-
-      if [[ -z ${last_fhr:-} ]]; then
-        last_fhr=${fhr}
-        continue
-      fi
+  case ${RUN} in
+    *gfs|gefs|sfs) # Link output files for RUN=gfs|enkfgfs|gefs|sfs
```

Comment:
```suggestion
    gfs|enkfgfs|gefs|sfs) # Link output files for RUN=gfs|enkfgfs|gefs|sfs
```
Making this explicit would also make it clear.  Thoughts?

Reply:
I'm fine with being explicit

Reply:
Done